ddsrecorder_yaml/include/ddsrecorder_yaml/replayer/CommandlineArgsReplayer.hpp

```diff
@@ -0,0 +1,61 @@
+// Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <string>
+
+#include <cpp_utils/types/Fuzzy.hpp>
+
+#include <ddspipe_core/configuration/CommandlineArgs.hpp>
+
+#include <ddspipe_core/library/library_dll.h>
+
+namespace eprosima {
+namespace ddsrecorder {
```

Comment:
Please add the replayer namespace.

```
namespace eprosima {
namespace ddsrecorder {
namespace replayer {
```

Reply:
There's no eprosima::ddsrecorder::replayer::yaml namespace, everything related with CommandlineArgsReplayer is in eprosima::ddsrecorder::yaml.